### PR TITLE
[SPARK-29308][BUILD] Update deps in dev/deps/spark-deps-hadoop-3.2 for hadoop-3.2

### PR DESCRIPTION
**Pull request by AngersZhuuuu**
### What changes were proposed in this pull request?

Current dev/deps/spark-deps-hadoop-3.2 have some wrong deps,   it's caused by `dev/test-dependencies.sh ` when build assembly dependencies. 
add maven compile parameter `-am` to make it build with all deps, and get right result. 

And update NOTICE-binary & NOTICE-binary for updated result.

### Why are the changes needed?
Update dev/deps/spark-hadoop-3.2

### Does this PR introduce any user-facing change?
No

### How was this patch tested?
N/A

### CODE REVIEW

**Comment by dongjoon-hyun** [comment]
cc @wangyum

**Comment by wangyum** [comment]
Thank you @dongjoon-hyun for ping me. @AngersZhuuuu We need to fix `dev/test-dependencies.sh`. The manifest generated by `./dev/test-dependencies.sh --replace-manifest`.

**Comment by AngersZhuuuu** [comment]
> Thank you @dongjoon-hyun for ping me. @AngersZhuuuu We need to fix `dev/test-dependencies.sh`. The manifest generated by `./dev/test-dependencies.sh --replace-manifest`.

Checking this, will remove WIP after fix this problem.

**Comment by AngersZhuuuu** [comment]
In sql/core module, in pom have this, and this module's build-classpath isright.
```
    <dependency>
      <groupId>org.apache.orc</groupId>
      <artifactId>orc-core</artifactId>
      <classifier>${orc.classifier}</classifier>
    </dependency>
    <dependency>
      <groupId>org.apache.orc</groupId>
      <artifactId>orc-mapreduce</artifactId>
      <classifier>${orc.classifier}</classifier>
    </dependency>
```


Get  `dependency:build-classpath` of parent pom, it will get correct orc jar path(not **nohive**)  for all module.
If we get `dependency:build-classpath` for sub-module，  it will get wrong  orc jar path(**nohive**). 

But profile `hadoop-3.2` truly work, for zookeeper jar in hadoop-2.7 version is 3.4.6, in hadoop-3.2 version is 3.4.13, it truly works when we get `dependency:build-classpath` of assembly with profile `hadoop-3.2`. 



cc @wangyum @srowen @dongjoon-hyun 
So I change `dev/test-dependencies.sh` to use `-am` with `-pl assembly` to get build result. 
See the change in `dep/deps/spark-deps-hadoop-3.2`
I have check all change, seem all reasonable.

**Comment by srowen** [comment]
Does this `-am` change affect the output of the Hadoop 2.7 deps?

I agree it sounds like this is a better way to identify what JARs are in the published binary release, as that is what the licenses must reflect. Is it possible to build the Hadoop 3.2 release and spot check whether the changes are correct -- removed deps really don't end up in the jars/ folder, and new deps do?

If there are real changes to 3.2 or 2.7 deps, we need to also reevaluate what is in our LICENSE-binary / license-binary/ files to ensure we have the right statements.

**Comment by AngersZhuuuu** [comment]
> Does this `-am` change affect the output of the Hadoop 2.7 deps?

Won't change hadoop-2.7. 


>  Is it possible to build the Hadoop 3.2 release and spot check whether the changes are correct.

Checked project deps in IntelliJIDEAL, it truly removed and new dep added. I will build  a hadoop-3.2 release to double check.

> If there are real changes to 3.2 or 2.7 deps, we need to also reevaluate what is in our LICENSE-binary / license-binary/ files to ensure we have the right statements.

Have considered about this. Since `-am` won't change `spark-deps-hadoop-2.7 's output. So we just need to check changes about new added in `spark-deps-hadoop-3.2`

**Comment by AngersZhuuuu** [comment]
@srowen 
There are some hive jars added with hive group
```
hive-beeline-2.3.6.jar
hive-cli-2.3.6.jar
hive-common-2.3.6.jar
hive-exec-2.3.6-core.jar
hive-jdbc-2.3.6.jar
hive-llap-common-2.3.6.jar
hive-metastore-2.3.6.jar
hive-serde-2.3.6.jar
hive-service-rpc-2.3.6.jar
hive-shims-0.23-2.3.6.jar
hive-shims-2.3.6.jar
hive-shims-common-2.3.6.jar
hive-shims-scheduler-2.3.6.jar
hive-storage-api-2.6.0.jar
hive-vector-code-gen-2.3.6.jar
```
 since origin LICENSE-binary add  `org.spark-project.hive` group hive jars. 
In hadoop-3.2, we add jars in `org.apache.hive` group, I think we should update this too, what do you think?

**Comment by AngersZhuuuu** [comment]
> Is it possible to build the Hadoop 3.2 release and spot check whether the changes are correct -- removed deps really don't end up in the jars/ folder, and new deps do?

@srowen  Build a hADOOP 3.2 release and checked it. Removed deps don't show in jars/ folder and new deps in the folder.

**Review by srowen — CHANGES_REQUESTED** [review]
Great work! I verified the changes and looks like you got a few tricky cases right. I have a few more suggestions to update the license / notice info. Hadoop 3.2's situation is still all but identical to 2.7, so I don't think we have to 'fork' them at this point, esp as 2.7 support will probably go away soon.

1. I'd remove the `org.spark-project` listings in `LICENSE-binary`. It's redundant with `org.apache.hive` listings below it; they're really straight renames of the binary artifacts.
2. New dependencies like `org.apache.hive:hive-beeline` etc need to be listed in `LICENSE-binary`
3. This is a little annoying, but I'd check the new Hive dependencies for a `NOTICE` file in each and add them to `NOTICE-binary` where they don't already exist, along with the existing Hive `NOTICE` entries.
4. I think we need to add a line for velocity in `LICENSE-binary`? It's already in `NOTICE-binary`, so not sure what happened to the former entry

File: dev/test-dependencies.sh [hunk 1]
```diff
@@ -75,7 +75,7 @@ for HADOOP_PROFILE in "${HADOOP_PROFILES[@]}"; do
 
   echo "Generating dependency manifest for $HADOOP_PROFILE"
   mkdir -p dev/pr-deps
-  $MVN $HADOOP2_MODULE_PROFILES -P$HADOOP_PROFILE dependency:build-classpath -pl assembly \
+  $MVN $HADOOP2_MODULE_PROFILES -P$HADOOP_PROFILE dependency:build-classpath -pl assembly -am \
```
Comment (dongjoon-hyun) [inline dev/test-dependencies.sh]: Oh, nice.

**Comment by dongjoon-hyun** [comment]
It seems that we need to change `branch-2.4`, too.

**Comment by srowen** [comment]
Yes to branch-2.4 too. I'd do that separately as I expect (hope?) there is little additional change compared to what the new Hadoop 3.2 profile might have presented here.

**Comment by dongjoon-hyun** [comment]
ok to test

**Comment by dongjoon-hyun** [comment]
And, @AngersZhuuuu . Please update the PR description.

File: dev/deps/spark-deps-hadoop-3.2 [hunk 2]
```diff
@@ -79,7 +78,21 @@ hadoop-yarn-common-3.2.0.jar
...
+hive-exec-2.3.6-core.jar
+hive-jdbc-2.3.6.jar
+hive-llap-common-2.3.6.jar
+hive-metastore-2.3.6.jar
+hive-serde-2.3.6.jar
+hive-service-rpc-2.3.6.jar
+hive-shims-0.23-2.3.6.jar
+hive-shims-2.3.6.jar
+hive-shims-common-2.3.6.jar
+hive-shims-scheduler-2.3.6.jar
```
Comment (dongjoon-hyun) [inline dev/deps/spark-deps-hadoop-3.2]: cc @wangyum .

**Comment by AngersZhuuuu** [comment]
> 2. New dependencies like org.apache.hive:hive-beeline etc need to be listed in LICENSE-binary

Have done this and checked other changes.

> 3. This is a little annoying, but I'd check the new Hive dependencies for a NOTICE file in each and add them to NOTICE-binary where they don't already exist, along with the existing Hive NOTICE entries.

I will check this today

> 4. I think we need to add a line for velocity in `LICENSE-binary`? It's already in `NOTICE-binary`, so not sure what happened to the former entry

Not very clear what you mean.

File: dev/deps/spark-deps-hadoop-3.2 (hunk 2 above)
Comment (wangyum) [reply dev/deps/spark-deps-hadoop-3.2]: Thank you @AngersZhuuuu @dongjoon-hyun It is exactly match.

File: LICENSE-binary [hunk 3]
```diff
@@ -369,11 +369,6 @@ org.datanucleus:datanucleus-api-jdo
 org.datanucleus:datanucleus-core
 org.datanucleus:datanucleus-rdbms
 org.lz4:lz4-java
-org.spark-project.hive:hive-beeline
-org.spark-project.hive:hive-cli
-org.spark-project.hive:hive-exec
-org.spark-project.hive:hive-jdbc
-org.spark-project.hive:hive-metastore
```
Comment (AngersZhuuuu) [inline LICENSE-binary]: cc @srowen

File: NOTICE-binary [hunk 4]
```diff
@@ -1140,18 +1140,36 @@ limitations under the License.
...
+
 Hive Serde
 Copyright 2019 The Apache Software Foundation
 
 Hive Service RPC
 Copyright 2019 The Apache Software Foundation
 
+Hive Shims
+Copyright 2019 The Apache Software Foundation
+
```
Comment (AngersZhuuuu) [inline NOTICE-binary]: @srowen  Hive jar notice added

**Comment by wangyum** [comment]
retest this please

**Comment by srowen** [comment]
Re: Velocity, I mean that it appears to be a dependency, but is not listed in `LICENSE-binary`. It needs an entry in there somewhere.

Re: Hive, it is looking good now. What's that question about org.apache.hive?

**Comment by AngersZhuuuu** [comment]
> Re: Velocity, I mean that it appears to be a dependency, but is not listed in `LICENSE-binary`. It needs an entry in there somewhere.
> 
> Re: Hive, it is looking good now. What's that question about org.apache.hive?


For Velocity, it has been in LICENSE-binary,  https://github.com/apache/spark/blob/de360e96d776f51c0fd7c31dcec142feabf3d543/LICENSE-binary#L411

For hive , I think it's ok, seems no more need to supplement.

**Comment by srowen** [comment]
Oh OK good I missed that. I think this looks pretty OK. Are there any more changes you want to make?

**Review by srowen — APPROVED** [review]

**Comment by AngersZhuuuu** [comment]
> Oh OK good I missed that. I think this looks pretty OK. Are there any more changes you want to make?

Need to check if any hadoop 3.2 deps jars have been added to LICENSE-binary. 
I'm not sure if the previous changes will leave such problems since previous spark-deps-hadoop-3.2 is not correct. I will check this.

**Comment by AngersZhuuuu** [comment]
@srowen  I have updated some LICENSE-binary, some jar I can't find's notice. 
If you knew could you provide some solution?

File: LICENSE-binary [hunk 5]
```diff
@@ -226,6 +226,7 @@ org.scalanlp:breeze_2.12
 org.typelevel:macro-compat_2.12
 org.yaml:snakeyaml
 org.apache.xbean:xbean-asm5-shaded
```
Comment (srowen) [inline LICENSE-binary]: I'm pretty sure you can delete this as obsolete, while we're at it. We're on ASM7.

File: LICENSE-binary [hunk 6]
```diff
@@ -320,11 +322,14 @@ com.github.mifmif:generex
 com.github.stephenc.jcip:jcip-annotations
 com.google.code.findbugs:jsr305
 com.google.code.gson:gson
+com.google.flatbuffers:flatbuffers-java
+com.google.guava:guava
```
Comment (srowen) [inline LICENSE-binary]: Do we really bring in unshaded Guava?

File: NOTICE-binary [hunk 7]
```diff
@@ -68,6 +68,9 @@ Copyright 2006-2013 Joe Walnes, Henri Tremblay, Leonardo Mesquita
 Apache XBean :: ASM 5 shaded (repackaged)
```
Comment (srowen) [inline NOTICE-binary]: Also delete this

File: LICENSE-binary (hunk 6 above)
Comment (srowen) [reply LICENSE-binary]: Wait, where are all these additional license entries coming from? according to the deps file, the assembly doesn't bring them in. Are you seeing these in the distro jars/ folder?

**Review by srowen — COMMENTED** [review]
If an artifact has no NOTICE file, there's nothing to do. That's fine.

File: LICENSE-binary (hunk 6 above)
Comment (AngersZhuuuu) [reply LICENSE-binary]: > Wait, where are all these additional license entries coming from? according to the deps file, the assembly doesn't bring them in. Are you seeing these in the distro jars/ folder?

Yes, it bring guava in dist/jars/

Comment (srowen) [reply LICENSE-binary]: OK. BTW do you have the output of `ls jars/` on the distro? just for reference here.

File: LICENSE-binary [hunk 8]
```diff
@@ -516,6 +522,7 @@ Common Development and Distribution License (CDDL) 1.1
 
 javax.el:javax.el-api	https://javaee.github.io/uel-ri/
 javax.servlet:javax.servlet-api   https://javaee.github.io/servlet-spec/
+javax.servlet.jsp:jsp-api
```
Comment (srowen) [inline LICENSE-binary]: For super completeness, let's put a copy of its license in `license-binary/`

**Review by srowen — CHANGES_REQUESTED** [review]

File: LICENSE-binary (hunk 6 above)
Comment (AngersZhuuuu) [reply LICENSE-binary]:
```
accessors-smart-1.2.jar
activation-1.1.1.jar
aircompressor-0.10.jar
algebra_2.12-2.0.0-M2.jar
antlr4-runtime-4.7.1.jar
antlr-runtime-3.5.2.jar
aopalliance-1.0.jar
aopalliance-repackaged-2.5.0.jar
arpack_combined_all-0.1.jar
arrow-format-0.12.0.jar
arrow-memory-0.12.0.jar
arrow-vector-0.12.0.jar
audience-annotations-0.5.0.jar
automaton-1.11-8.jar
avro-1.8.2.jar
avro-ipc-1.8.2.jar
avro-mapred-1.8.2-hadoop2.jar
bonecp-0.8.0.RELEASE.jar
breeze-macros_2.12-1.0.jar
breeze_2.12-1.0.jar
cats-kernel_2.12-2.0.0-M4.jar
chill-java-0.9.3.jar
chill_2.12-0.9.3.jar
commons-beanutils-1.9.3.jar
commons-cli-1.2.jar
commons-codec-1.10.jar
commons-collections-3.2.2.jar
commons-compiler-3.0.15.jar
commons-compress-1.8.1.jar
commons-configuration2-2.1.1.jar
commons-crypto-1.0.0.jar
commons-daemon-1.0.13.jar
commons-dbcp-1.4.jar
commons-httpclient-3.1.jar
commons-io-2.4.jar
commons-lang3-3.8.1.jar
commons-lang-2.6.jar
commons-logging-1.1.3.jar
commons-math3-3.4.1.jar
commons-net-3.1.jar
commons-pool-1.5.4.jar
commons-text-1.6.jar
compress-lzf-1.0.3.jar
core-1.1.2.jar
curator-client-2.13.0.jar
curator-framework-2.13.0.jar
curator-recipes-2.13.0.jar
datanucleus-api-jdo-4.2.4.jar
datanucleus-core-4.1.17.jar
datanucleus-rdbms-4.1.19.jar
derby-10.12.1.1.jar
dnsjava-2.1.7.jar
dropwizard-metrics-hadoop-metrics2-reporter-0.1.2.jar
ehcache-3.3.1.jar
flatbuffers-java-1.9.0.jar
generex-1.0.2.jar
geronimo-jcache_1.0_spec-1.0-alpha-1.jar
gson-2.2.4.jar
guava-14.0.1.jar
guice-4.0.jar
guice-servlet-4.0.jar
hadoop-annotations-3.2.0.jar
hadoop-auth-3.2.0.jar
hadoop-client-3.2.0.jar
hadoop-common-3.2.0.jar
hadoop-hdfs-client-3.2.0.jar
hadoop-mapreduce-client-common-3.2.0.jar
hadoop-mapreduce-client-core-3.2.0.jar
hadoop-mapreduce-client-jobclient-3.2.0.jar
hadoop-yarn-api-3.2.0.jar
hadoop-yarn-client-3.2.0.jar
hadoop-yarn-common-3.2.0.jar
hadoop-yarn-registry-3.2.0.jar
hadoop-yarn-server-common-3.2.0.jar
hadoop-yarn-server-web-proxy-3.2.0.jar
HikariCP-2.5.1.jar
hive-beeline-2.3.6.jar
hive-cli-2.3.6.jar
hive-common-2.3.6.jar
hive-exec-2.3.6-core.jar
hive-jdbc-2.3.6.jar
hive-llap-common-2.3.6.jar
hive-metastore-2.3.6.jar
hive-serde-2.3.6.jar
hive-service-rpc-2.3.6.jar
hive-shims-0.23-2.3.6.jar
hive-shims-2.3.6.jar
hive-shims-common-2.3.6.jar
hive-shims-scheduler-2.3.6.jar
hive-storage-api-2.6.0.jar
hive-vector-code-gen-2.3.6.jar
hk2-api-2.5.0.jar
hk2-locator-2.5.0.jar
hk2-utils-2.5.0.jar
hppc-0.7.2.jar
htrace-core4-4.1.0-incubating.jar
httpclient-4.5.6.jar
httpcore-4.4.10.jar
istack-commons-runtime-3.0.8.jar
ivy-2.4.0.jar
jackson-annotations-2.9.10.jar
jackson-core-2.9.10.jar
jackson-core-asl-1.9.13.jar
jackson-databind-2.9.10.jar
jackson-dataformat-yaml-2.9.10.jar
jackson-jaxrs-base-2.9.5.jar
jackson-jaxrs-json-provider-2.9.5.jar
jackson-mapper-asl-1.9.13.jar
jackson-module-jaxb-annotations-2.9.10.jar
jackson-module-paranamer-2.9.10.jar
jackson-module-scala_2.12-2.9.10.jar
jakarta.annotation-api-1.3.4.jar
jakarta.inject-2.5.0.jar
jakarta.ws.rs-api-2.1.5.jar
jakarta.xml.bind-api-2.3.2.jar
janino-3.0.15.jar
javassist-3.22.0-CR2.jar
javax.el-3.0.1-b11.jar
javax.inject-1.jar
javax.jdo-3.2.0-m3.jar
javax.servlet-api-3.1.0.jar
javolution-5.5.1.jar
jaxb-api-2.2.11.jar
jaxb-runtime-2.3.2.jar
jcip-annotations-1.0-1.jar
jcl-over-slf4j-1.7.16.jar
jdo-api-3.0.1.jar
jersey-client-2.29.jar
jersey-common-2.29.jar
jersey-container-servlet-2.29.jar
jersey-container-servlet-core-2.29.jar
jersey-hk2-2.29.jar
jersey-media-jaxb-2.29.jar
jersey-server-2.29.jar
jetty-webapp-9.4.18.v20190429.jar
jetty-xml-9.4.18.v20190429.jar
JLargeArrays-1.5.jar
jline-2.14.6.jar
joda-time-2.9.3.jar
jodd-core-3.5.2.jar
jpam-1.1.jar
json4s-ast_2.12-3.6.6.jar
json4s-core_2.12-3.6.6.jar
json4s-jackson_2.12-3.6.6.jar
json4s-scalap_2.12-3.6.6.jar
json-1.8.jar
json-smart-2.3.jar
jsp-api-2.1.jar
jsr305-3.0.0.jar
jta-1.1.jar
JTransforms-3.1.jar
jul-to-slf4j-1.7.16.jar
kerb-admin-1.0.1.jar
kerb-client-1.0.1.jar
kerb-common-1.0.1.jar
kerb-core-1.0.1.jar
kerb-crypto-1.0.1.jar
kerb-identity-1.0.1.jar
kerb-server-1.0.1.jar
kerb-simplekdc-1.0.1.jar
kerb-util-1.0.1.jar
kerby-asn1-1.0.1.jar
kerby-config-1.0.1.jar
kerby-pkix-1.0.1.jar
kerby-util-1.0.1.jar
kerby-xdr-1.0.1.jar
kryo-shaded-4.0.2.jar
kubernetes-client-4.4.2.jar
kubernetes-model-4.4.2.jar
kubernetes-model-common-4.4.2.jar
leveldbjni-all-1.8.jar
libfb303-0.9.3.jar
libthrift-0.12.0.jar
log4j-1.2.17.jar
logging-interceptor-3.12.0.jar
lz4-java-1.6.0.jar
machinist_2.12-0.6.8.jar
macro-compat_2.12-1.1.1.jar
mesos-1.4.0-shaded-protobuf.jar
metrics-core-3.1.5.jar
metrics-graphite-3.1.5.jar
metrics-json-3.1.5.jar
metrics-jvm-3.1.5.jar
minlog-1.3.0.jar
mssql-jdbc-6.2.1.jre7.jar
netty-all-4.1.39.Final.jar
nimbus-jose-jwt-4.41.1.jar
objenesis-2.5.1.jar
okapi-shade-0.4.2.jar
okhttp-2.7.5.jar
okhttp-3.8.1.jar
okio-1.13.0.jar
opencsv-2.3.jar
orc-core-1.5.6.jar
orc-mapreduce-1.5.6.jar
orc-shims-1.5.6.jar
oro-2.0.8.jar
osgi-resource-locator-1.0.3.jar
paranamer-2.8.jar
parquet-column-1.10.1.jar
parquet-common-1.10.1.jar
parquet-encoding-1.10.1.jar
parquet-format-2.4.0.jar
parquet-hadoop-1.10.1.jar
parquet-jackson-1.10.1.jar
protobuf-java-2.5.0.jar
py4j-0.10.8.1.jar
pyrolite-4.30.jar
re2j-1.1.jar
RoaringBitmap-0.7.45.jar
scala-collection-compat_2.12-2.1.1.jar
scala-compiler-2.12.10.jar
scala-library-2.12.10.jar
scala-parser-combinators_2.12-1.1.2.jar
scala-reflect-2.12.10.jar
scala-xml_2.12-1.2.0.jar
shapeless_2.12-2.3.3.jar
shims-0.7.45.jar
slf4j-api-1.7.16.jar
slf4j-log4j12-1.7.16.jar
snakeyaml-1.23.jar
snappy-java-1.1.7.3.jar
spark-catalyst_2.12-3.0.0-SNAPSHOT.jar
spark-core_2.12-3.0.0-SNAPSHOT.jar
spark-cypher_2.12-3.0.0-SNAPSHOT.jar
spark-graph-api_2.12-3.0.0-SNAPSHOT.jar
spark-graph_2.12-3.0.0-SNAPSHOT.jar
spark-graphx_2.12-3.0.0-SNAPSHOT.jar
spark-hive-thriftserver_2.12-3.0.0-SNAPSHOT.jar
spark-hive_2.12-3.0.0-SNAPSHOT.jar
spark-kubernetes_2.12-3.0.0-SNAPSHOT.jar
spark-kvstore_2.12-3.0.0-SNAPSHOT.jar
spark-launcher_2.12-3.0.0-SNAPSHOT.jar
spark-mesos_2.12-3.0.0-SNAPSHOT.jar
spark-mllib-local_2.12-3.0.0-SNAPSHOT.jar
spark-mllib_2.12-3.0.0-SNAPSHOT.jar
spark-network-common_2.12-3.0.0-SNAPSHOT.jar
spark-network-shuffle_2.12-3.0.0-SNAPSHOT.jar
spark-repl_2.12-3.0.0-SNAPSHOT.jar
spark-sketch_2.12-3.0.0-SNAPSHOT.jar
spark-sql_2.12-3.0.0-SNAPSHOT.jar
spark-streaming_2.12-3.0.0-SNAPSHOT.jar
spark-tags_2.12-3.0.0-SNAPSHOT.jar
spark-tags_2.12-3.0.0-SNAPSHOT-tests.jar
spark-unsafe_2.12-3.0.0-SNAPSHOT.jar
spark-yarn_2.12-3.0.0-SNAPSHOT.jar
spire-macros_2.12-0.17.0-M1.jar
spire-platform_2.12-0.17.0-M1.jar
spire-util_2.12-0.17.0-M1.jar
spire_2.12-0.17.0-M1.jar
ST4-4.0.4.jar
stax2-api-3.1.4.jar
stax-api-1.0.1.jar
stream-2.9.6.jar
super-csv-2.2.0.jar
token-provider-1.0.1.jar
transaction-api-1.1.jar
univocity-parsers-2.7.3.jar
validation-api-2.0.1.Final.jar
velocity-1.5.jar
woodstox-core-5.0.3.jar
xbean-asm7-shaded-4.14.jar
xz-1.5.jar
zjsonpatch-0.3.0.jar
zookeeper-3.4.14.jar
zstd-jni-1.4.2-1.jar
```

File: LICENSE-binary (hunk 8 above)
Comment (AngersZhuuuu) [reply LICENSE-binary]: > For super completeness

Add this 
https://github.com/javaee/javaee-jsp-api/blob/master/LICENSE

**Comment by AngersZhuuuu** [comment]
@dongjoon-hyun 
test failed but seems not my error. Could you help to trigger again?

**Comment by wangyum** [comment]
retest this please

**Comment by dongjoon-hyun** [comment]
Unfortunately, Jenkins is down. Let's wait until Jenkins is back.

**Comment by AngersZhuuuu** [comment]
> Unfortunately, Jenkins is down. Let's wait until Jenkins is back.


> Unfortunately, Jenkins is down. Let's wait until Jenkins is back.

Bad news, wait for Jenkins back.

**Comment by wangyum** [comment]
retest this please

File: licenses-binary/LICENSE-javax-transaction-transaction-api.txt [hunk 9]
```diff
@@ -0,0 +1,119 @@
...
+
+C. Any new file that is contributed or otherwise made available under the terms of this License.
+
+1.10. Original Software means the Source Code and Executable form of computer software code that is originally released under this License.
+
+1.11. Patent Claims means any patent claim(s), now owned or hereafter acquired, including without limitation, method, process, and apparatus claims, in any patent Licensable by grantor.
+
+1.12. Source Code means (a) the common form of computer software code in which modifications are made and (b) associated documentation included in or with such code.
+
+1.13. You (or Your) means an individual or a legal entity exercising rights under, and complying with all of the terms of, this License. For legal entities, You includes any entity which controls, is controlled by, or is under common control with You. For purposes of this definition, control means (a)�the power, direct or indirect, to cause the direction or management of such entity, whether by contract or otherwise, or (b)�ownership of more than fifty percent (50%) of the outstanding shares or beneficial ownership of such entity.
```
Comment (dongjoon-hyun) [inline licenses-binary/LICENSE-javax-transaction-transaction-api.txt]: The original license has a non-ascii char?

Comment (srowen) [reply licenses-binary/LICENSE-javax-transaction-transaction-api.txt]: Yeah I noticed that. I don't think it's material, but, if there were a cleaner plain-text license, sure, would be better.

Comment (AngersZhuuuu) [reply licenses-binary/LICENSE-javax-transaction-transaction-api.txt]: > The original license has a non-ascii char?

In my local file I didn't see � . Strange things..

Comment (AngersZhuuuu) [reply licenses-binary/LICENSE-javax-transaction-transaction-api.txt]: > The original license has a non-ascii char?

Remove non-ascii char by hand.

**Review by srowen — APPROVED** [review]

**Comment by srowen** [comment]
Merged to master

**Pull request closed without merging**